pkgs/by-name/fr/freecad/package.nix

```diff
@@ -54,144 +58,151 @@ let
-    longDescription = ''
-      FreeCAD is an open-source parametric 3D modeler made primarily to design
-      real-life objects of any size. Parametric modeling allows you to easily
-      modify your design by going back into your model history and changing its
-      parameters.
+    passthru.tests = {
+      # Check that things such as argument parsing still work correctly with
+      # the above PYTHONPATH patch. Previously the patch used above changed
+      # the `PyConfig_InitIsolatedConfig` to `PyConfig_InitPythonConfig`,
+      # which caused the built-in interpreter to attempt (and fail) to doubly
+      # parse argv. This should catch if that ever regresses and also ensures
+      # that PYTHONPATH is still respected enough for the FreeCAD console to
+      # successfully run and check that it was included in `sys.path`.
+      python-path =
+        runCommand "freecad-test-console" {
+          nativeBuildInputs = [freecad];
+        } ''
+          HOME="$(mktemp -d)" PYTHONPATH="$(pwd)/test" FreeCADCmd --log-file $out -c "if not '$(pwd)/test' in sys.path: sys.exit(1)" </dev/null
+        '';
+    };
 
-      FreeCAD allows you to sketch geometry constrained 2D shapes and use them
-      as a base to build other objects. It contains many components to adjust
-      dimensions or extract design details from 3D models to create high quality
-      production ready drawings.
+    meta = {
+      homepage = "https://www.freecad.org";
+      description = "General purpose Open Source 3D CAD/MCAD/CAx/CAE/PLM modeler";
+      longDescription = ''
+        FreeCAD is an open-source parametric 3D modeler made primarily to design
+        real-life objects of any size. Parametric modeling allows you to easily
+        modify your design by going back into your model history and changing its
+        parameters.
 
-      FreeCAD is designed to fit a wide range of uses including product design,
-      mechanical engineering and architecture. Whether you are a hobbyist, a
-      programmer, an experienced CAD user, a student or a teacher, you will feel
-      right at home with FreeCAD.
-    '';
-    license = lib.licenses.lgpl2Plus;
-    maintainers = with lib.maintainers; [ viric gebner AndersonTorres ];
-    platforms = lib.platforms.linux;
-  };
-})
+        FreeCAD allows you to sketch geometry constrained 2D shapes and use them
+        as a base to build other objects. It contains many components to adjust
+        dimensions or extract design details from 3D models to create high quality
+        production ready drawings.
+
+        FreeCAD is designed to fit a wide range of uses including product design,
+        mechanical engineering and architecture. Whether you are a hobbyist, a
+        programmer, an experienced CAD user, a student or a teacher, you will feel
+        right at home with FreeCAD.
+      '';
+      license = lib.licenses.lgpl2Plus;
+      maintainers = with lib.maintainers; [viric gebner AndersonTorres];
```

Comment:
What formatter are you using?

Reply:
Oops, sorry I missed that one, that was unintended! It appears to be the default formatter that `nvf` is using... I'll reconfig and revert that part...